src/bindfs.c

```diff
@@ -1643,7 +1873,7 @@ static struct fuse_operations bindfs_oper = {
     #ifndef HAVE_FUSE_3
     .fgetattr   = bindfs_fgetattr,
     #endif
-    /* no access() since we always use -o default_permissions */
+    .access     = bindfs_access,
```

Comment:
I guess the next major step is to add a call to the access check code to the other `bindfs_` functions. If I'm not mistaken, `bindfs_read`, `bindfs_write` and `bindfs_readdir` don't need it because the check should only be done on file/dir open. But almost all other functions will need it.

I'm ~95% sure that libfuse doesn't call `access` for us before each of these functions, but maybe it's worth checking.

It's probably best to move most of the code in `bindfs_access` into a function that takes a processed path, so `process_path` isn't called twice when the other functions call the permission checker.